hazelcast/src/main/java/com/hazelcast/map/impl/operation/steps/PutOpSteps.java

```diff
@@ -140,7 +140,7 @@ public void runStep(State state) {
         @Override
         public Step nextStep(State state) {
             if (state.isStopExecution()) {
-                return UtilSteps.SEND_RESPONSE;
+                return UtilSteps.FINAL_STEP;
```

Comment:
Is it correct that now for tiered store we always execute Put/GetOperation using Put/GetOpSteps mechanics? If do, where is it enabled?